### PR TITLE
Add new recipe Inspector

### PR DESCRIPTION
**Pull request by czc**
Inspector is a tool for long-read de novo assembly evaluation

https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02527-4

[Describe your pull request here
](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02527-4)
----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by czc** [comment]
@BiocondaBot please add label

File: recipes/inspector/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  url: https://github.com/ChongLab/Inspector/archive/refs/tags/inspector-v{{ version }}.tar.gz
+  sha256: fac61240ee7eff2ba0624491c6e5e050567564695cc79096f54d4e1bb66b8b35
+
+build:
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - python <3
```
Comment (bgruening) [inline recipes/inspector/meta.yaml:17]: Python 2 is end of life. Is there any hope we can get a Python3 version of this? For example, you can not use this tool in combination with any Python3 tool.

File: recipes/inspector/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  run:
+    - python <3
+    - minimap2 =2.15
```
Comment (bgruening) [inline recipes/inspector/meta.yaml:20]: Are those strict pins really necessary? Please consider removing the pins.

File: recipes/inspector/meta.yaml (hunk 1 above)
Comment (czc) [reply recipes/inspector/meta.yaml:17]: Same here. Thanks very much for reviewing and for the suggestion. The conda environment has been tested locally. The python3 version is planned to be updated. Could we release this build and upload future versions later? Thank you.

File: recipes/inspector/meta.yaml (hunk 2 above)
Comment (czc) [reply recipes/inspector/meta.yaml:20]: Strict to match python 2.7. Will try to remove them later. Thank you.

Comment (czc) [reply recipes/inspector/meta.yaml:20]: @bgruening  We have updated the codes and it supports Python 3 now. Could you help review and merge? Thank you so much in advance.

**Review by dpryan79 — APPROVED** [review]

**Pull request merged**